app/models/category_transaction.rb

```diff
@@ -0,0 +1,5 @@
+class CategoryTransaction < ApplicationRecord
+  belongs_to :category
+  belongs_to :my_transaction, class_name: 'Transaction'
+  # belongs_to :transaction
```

Comment:
Please remove commented code from here as it can make the whole project unclean.